compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/reference/FunctionReference.kt

```diff
@@ -85,7 +89,7 @@ public sealed class FunctionReference {
   public class Psi internal constructor(
     public val function: KtFunction,
     override val declaringClass: ClassReference.Psi,
-    override val fqName: FqName = function.fqName ?: declaringClass.fqName
```

Comment:
I accidentally committed that in my previous PR. This was wrong and I forgot to fix it.